### PR TITLE
issue-291: Fix schema to match mixs.yaml

### PR DESCRIPTION
**Pull request by wdduncan**
update branch issue-291

### CODE REVIEW

**Comment by wdduncan** [comment]
@cmungall I've generated new data based on the updated schema. The database validates.

File: schema/nmdc.yaml [hunk 1]
```diff
@@ -1333,6 +1165,15 @@ slots:
   sample_collection_minute:
     range: integer
     
+  salinity_category:
```
Comment (cmungall) [inline schema/nmdc.yaml:1168]: add a see_also to a mixs ticket requesting this

File: schema/nmdc.yaml [hunk 2]
```diff
@@ -1333,6 +1165,15 @@ slots:
...
+    description: >-
+      Categorcial description of the sample's salinity.
+      Examples: halophile, halotolerant, hypersaline, huryhaline
+    range: string
+      
+  soluble_iron_micromol:
```
Comment (cmungall) [inline schema/nmdc.yaml:1174]: ditto

File: schema/mappings/gold-to-mixs.sssom.tsv [hunk 3]
```diff
@@ -40,8 +43,8 @@ gold.vocab:soil_store_condition	soil_store_condition	skos:exactMatch	mixs:store_
 gold.vocab:soil_link_climate_info	soil_link_climate_info	skos:exactMatch	mixs:link_climate_info	link_climate_info	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:annual_temp	annual_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:season_temp	season_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:season_precpt	season_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:annual_precpt	annual_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
+gold.vocab:soil_annual_season_precpt	"soil_annual_season_precpt "	skos:closeMatch	mixs:season_precpt	season_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
```
Comment (cmungall) [inline schema/mappings/gold-to-mixs.sssom.tsv:46]: this is a mistake. Don't introduce spaces, and even if trailing spaces are required, don't quote in a TSV

File: schema/mappings/gold-to-mixs.sssom.tsv [hunk 4]
```diff
@@ -40,8 +43,8 @@ gold.vocab:soil_store_condition	soil_store_condition	skos:exactMatch	mixs:store_
 gold.vocab:soil_link_climate_info	soil_link_climate_info	skos:exactMatch	mixs:link_climate_info	link_climate_info	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:annual_temp	annual_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:season_temp	season_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:season_precpt	season_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:annual_precpt	annual_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
```
Comment (cmungall) [inline schema/mappings/gold-to-mixs.sssom.tsv:44]: why did you drop this one? it's a closeMatch so ok if not 1:1

**Review by cmungall — CHANGES_REQUESTED** [review]

File: schema/mappings/gold-to-mixs.sssom.tsv (hunk 3 above)
Comment (wdduncan) [reply schema/mappings/gold-to-mixs.sssom.tsv:46]: Not sure how the quotes got in there. Thanks for catching this!

File: schema/nmdc.yaml (hunk 1 above)
Comment (wdduncan) [reply schema/nmdc.yaml:1168]: Will do.

File: schema/mappings/gold-to-mixs.sssom.tsv (hunk 4 above)
Comment (wdduncan) [reply schema/mappings/gold-to-mixs.sssom.tsv:44]: I don't remember dropping these, but I must have done it ... I will add the mappings back. Right now the algorithm only looks exact matches (i.e., the gold fields are only mapped to exact matches).

**Review by cmungall — APPROVED** [review]

**Comment by cmungall** [comment]
as it should but let's keep closeMappings in when we don't have an exact

**Pull request merged**